example_configs/radicale.md

```diff
@@ -0,0 +1,27 @@
+#Configuration of RADICALE authentification with lldap.
+
+# See how to implement the correct plugin (v0.3) for radicale : https://github.com/shroomify-it/radicale-auth-ldap-plugin
```

Comment:
Hmm, rather than "See how to implement", how about: "Fork of the radicale LDAP plugin to work with LLDAP"

Reply:
And it shouldn't be a title (no #)